tests/opentelemetry/conftest.py

```diff
@@ -12,14 +10,6 @@
 
 
 @pytest.fixture
-def set_otel_python_context():
```

Comment:
By moving setting this envar to riot, we avoid an odd bug that popped up due to test config issues.